.github/workflows/flowzone.yml

```diff
@@ -554,7 +578,6 @@ jobs:
         run: |
           echo "not yet implemented!"
 
-      # TODO: publish to npmjs
```

Comment:
We don't need to support this flow for now, but it's nice to have for users with a "tag-master" worflow rather than finalizing on merge.